clients/ts/signalr/src/LongPollingTransport.ts

```diff
@@ -77,13 +71,13 @@ export class LongPollingTransport implements ITransport {
             this.logger.log(LogLevel.Error, `(LongPolling transport) Unexpected response code: ${response.statusCode}`);
 
             // Mark running as false so that the poll immediately ends and runs the close logic
-            closeError = new HttpError(response.statusText, response.statusCode);
+            this.closeError = new HttpError(response.statusText, response.statusCode);
             this.running = false;
         } else {
             this.running = true;
         }
 
-        this.poll(this.url, pollOptions, closeError);
+        this.receiving = this.poll(this.url, pollOptions);
         return Promise.resolve();
```

Comment:
What is this return for? Is it needed since this method has `async` on it?

Reply:
I don't think it's needed, an `async` function returns a promise already.